app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,20 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            skill_level: None
```

Comment:
The `skill_level` parameter should have a type hint that reflects its actual data type, such as `int`, instead of `None`.